jumpscale/tools/git/__init__.py

```diff
@@ -78,11 +78,15 @@ def clone_repo(url: str, dest: str, branch_or_tag="", depth=0, commit_id=""):
         cmd += f" -b {branch_or_tag}"
     if depth != 0:
         cmd += f" --depth={depth}"
-    j.core.executors.run_local(cmd)
+    rc, _, err = j.core.executors.run_local(cmd, warn=True)
+    if rc > 0:
+        raise j.exceptions.Runtime(err)
```

Comment:
please mention the command you were trying to run in the error message

Reply:
edited stderr :+1:, thanks